diff --git a/node_modules/@sentry/react-native/sentry.gradle b/node_modules/@sentry/react-native/sentry.gradle
index 587f8e0..942330b 100644
--- a/node_modules/@sentry/react-native/sentry.gradle
+++ b/node_modules/@sentry/react-native/sentry.gradle
@@ -300,7 +300,12 @@ project.afterEvaluate {
 def resolveSentryReactNativeSDKPath(reactRoot) {
     def resolvedSentryPath = null
     try {
-        resolvedSentryPath = new File(["node", "--print", "require.resolve('@sentry/react-native/package.json')"].execute(null, rootDir).text.trim()).getParentFile();
+        resolvedSentryPath = providers.exec {
+            commandLine "node", "--print", "require.resolve('@sentry/react-native/package.json')"
+            workingDir rootDir
+        }.standardOutput.asText.map { path ->
+            new File(path.trim()).getParentFile()
+        }.get()
     } catch (Throwable ignored) {} // if the resolve fails we fallback to the default path
     def sentryPackage = resolvedSentryPath != null && resolvedSentryPath.exists() ? resolvedSentryPath.getAbsolutePath() : "$reactRoot/node_modules/@sentry/react-native"
     return sentryPackage
