diff --git a/node_modules/expo-sqlite/android/build.gradle b/node_modules/expo-sqlite/android/build.gradle
index 810ebe7..326b75e 100644
--- a/node_modules/expo-sqlite/android/build.gradle
+++ b/node_modules/expo-sqlite/android/build.gradle
@@ -73,6 +73,12 @@ android {
   buildFeatures {
     prefab true
   }
+  packagingOptions {
+    excludes += [
+      "**/libc++_shared.so",
+      "**/libfbjni.so",
+     ]
+  }
 }
 
 dependencies {
