diff --git a/node_modules/expo-dev-client/android/build.gradle b/node_modules/expo-dev-client/android/build.gradle
index c7f6a4e..f345e61 100644
--- a/node_modules/expo-dev-client/android/build.gradle
+++ b/node_modules/expo-dev-client/android/build.gradle
@@ -35,6 +35,13 @@ android {
     buildConfig true
     viewBinding true
   }
+
+  packagingOptions {
+    merges += [
+      'META-INF/LICENSE.md',
+      'META-INF/LICENSE-notice.md'
+     ]
+  }
 }
 
 dependencies {
