diff --git a/node_modules/react-native-svg/android/build.gradle b/node_modules/react-native-svg/android/build.gradle
index c02400f..103bf36 100644
--- a/node_modules/react-native-svg/android/build.gradle
+++ b/node_modules/react-native-svg/android/build.gradle
@@ -44,7 +44,12 @@ def resolveReactNativeDirectory() {
   }
 
   // Fallback to node resolver for custom directory structures like monorepos.
-  def reactNativePackage = file(["node", "--print", "require.resolve('react-native/package.json')"].execute(null, rootDir).text.trim())
+  def reactNativePackage = providers.exec {
+    commandLine "node", "--print", "require.resolve('react-native/package.json')"
+    workingDir rootDir
+  }.standardOutput.asText.map { path ->
+    file(path.trim())
+  }.get()
   if (reactNativePackage.exists()) {
     return reactNativePackage.parentFile
   }
