diff --git a/node_modules/react-native-gesture-handler/android/build.gradle b/node_modules/react-native-gesture-handler/android/build.gradle
index 81e011b..7418112 100644
--- a/node_modules/react-native-gesture-handler/android/build.gradle
+++ b/node_modules/react-native-gesture-handler/android/build.gradle
@@ -39,7 +39,12 @@ def resolveReactNativeDirectory() {
     }
 
     // Fallback to node resolver for custom directory structures like monorepos.
-    def reactNativePackage = file(["node", "--print", "require.resolve('react-native/package.json')"].execute(null, rootDir).text.trim())
+    def reactNativePackage = providers.exec {
+        commandLine "node", "--print", "require.resolve('react-native/package.json')"
+        workingDir rootDir
+    }.standardOutput.asText.map { path ->
+        file(path.trim())
+    }.get()
     if (reactNativePackage.exists()) {
         return reactNativePackage.parentFile
     }
