# Adds a mock for `useReducedMotion`.

diff --git a/node_modules/react-native-reanimated/src/mock.ts b/node_modules/react-native-reanimated/src/mock.ts
index a6ff0ce..415aac2 100644
--- a/node_modules/react-native-reanimated/src/mock.ts
+++ b/node_modules/react-native-reanimated/src/mock.ts
@@ -82,7 +82,7 @@ const hook = {
       },
     });
   },
-  // useReducedMotion: ADD ME IF NEEDED
+  useReducedMotion: NOOP,
   useAnimatedStyle: IMMEDIATE_CALLBACK_INVOCATION,
   useAnimatedGestureHandler: NOOP_FACTORY,
   useAnimatedReaction: NOOP,
